examples/ax25/bpsk.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python3
```

Comment:
This file was not included before. I think it shouldn't be included in the repository, since users can generate it from the GRC file.

Reply:
Yep, sorry will remove it and add a .gitignore file as well to prevent similar issues

Reply:
Removed and added .gitignore.